scripts/ddl2cpp

```diff
@@ -485,6 +494,19 @@ def setArgumentBool(s, bool_value):
     var_name = first_lower(re.sub("(\s|-|[0-9])(\S)", repl_func_for_args, s))
     globals()[var_name] = bool_value
 
+def loadExtendedTypesFile(filename):
```

Comment:
One can probably make an argument for not loading a python script but a csv or other text file. I am not very proficient with python and decided with going this route. It should be easy enough to update/extend at this function without having to touch the rest of the code again.

Reply:
Could you try to go the CSV route? Reading CSV is quite simple in python.

Reply:
Sure I can give it a bash and see what I get

Reply:
Do you have an idea or suggestion on how a CSV would look for this?
Maybe:
```
extendedDdlIntegerTypes, "one", "two"
```
I think this is what tripped me up when I considered the approach, I could not think of a nice other format

Reply:
I think I have something that can work, will create a new PR once it works